features/admin.applications.v1/components/settings/info.tsx

```diff
@@ -181,14 +187,14 @@ export const Info: FunctionComponent<InfoPropsInterface> = (
                                                     <Link
                                                         external={ false }
                                                         onClick={ () => {
-                                                            history.push(
-                                                                AppConstants.getPaths()
-                                                                    .get("APPLICATION_SIGN_IN_METHOD_EDIT")
-                                                                    .replace(":id", appId)
-                                                                    .replace(
-                                                                        ":tabName",
-                                                                        `#tab=${ PROTOCOLS_TAB_INDEX }`
-                                                                    )
+                                                            history.push({
+                                                                pathname: AppConstants.getPaths()
+                                                                    .get("APPLICATION_EDIT")
+                                                                    .replace(":id", appId),
+                                                                search:
+                                                                    `?${ ApplicationManagementConstants.IS_PROTOCOL }=
+                                                                    true`
```

Comment:
Since template strings tend to preserve the white spaces, this will be transformed to something like 

```
?isProtocol=                                             true
```

to avoid this, let's use the good ol' string concatenation here.

```suggestion
                                                                search: "?" + ApplicationManagementConstants.IS_PROTOCOL +
                                                                    "=true"
```

Reply:
Fixed with 66db63647fd006555a02f3781c73a0c098394830